ddsp/training/docker/user_interaction.py

```diff
@@ -1,84 +1,235 @@
+                                 'from before training', skip=True))
+      if restore_dir == '':
+        restore_dir = save_dir
+  restore_dir = check_bucket(restore_dir, prompt('path', 'checkpoints will be '
+                                                 'restored from before '
+                                                 'training', skip=True))
   input_dict['restore_dir'] = restore_dir
 
-  config_path = input('\nInsert the path to a configuration file '
-                      'or skip for the default one:\n')
-  if config_path == '':
-    config_path = './config_single_vm.yaml'
+  if FLAGS.config_path:
+    config_path = FLAGS.config_path
+  else:
+    config_path = input(prompt('path', 'configuration file is stored',\
+                               skip=True, local=True))
+    if config_path == '':
+      config_path = './config_multiple_vms.yaml'
   input_dict['config_path'] = config_path
 
-  image_uri = input('\nInsert an IMAGE URI. '
-                    'The template is: gcr.io/<GCP_PROJECT_ID>'
-                    '/<IMAGE_REPO_NAME>:<IMAGE_TAG> :\n')
+  if FLAGS.project_id:
+    project_id = FLAGS.project_id
+  else:
+    project_id = subprocess.getoutput('gcloud config get-value project')
+    if project_id == '(unset)':
+      project_id = input(prompt('value', 'your project ID'))
+    check_project_id = 'gcloud projects describe ' + project_id
+    while not subprocess.getoutput(check_project_id).startswith('createTime'):
+      print('Project ID not valid.')
+      project_id = input(prompt('value', 'your project ID'))
+      check_project_id = 'gcloud projects describe ' + project_id
+
+  image_uri = 'gcr.io/' + project_id + '/ddsp_training:train_job'
   input_dict['image_uri'] = image_uri
 
-  job_name = input('\nInsert a job name:\n')
+  job_name = 'training_job_' + str(int((datetime.datetime.now()\
+             - datetime.datetime(1970,1,1)).total_seconds()))
   input_dict['job_name'] = job_name
 
-  region = input('\nInsert the region you want to train '
-                  'your model in or skip for the default '
-                  'value (europe-west4): ')
-  if region == '':
-    region = 'europe-west4'
+  if FLAGS.region:
+    region = FLAGS.region
+  else:
+    region = subprocess.getoutput('gcloud config get-value compute/region')
+    if region == '(unset)':
+      region = input('\nInsert the region you want to train '
+                      'your model in or skip for the default '
+                      'value (europe-west4): ')
+      if region == '':
+        region = 'europe-west4'
   input_dict['region'] = region
 
-  batch_size = input('\nInsert batch size '
-                      'or skip for the default value (128): ')
-  if batch_size == '':
-    batch_size = '128'
+  if FLAGS.batch_size:
+    batch_size = FLAGS.batch_size
+  else:
+    batch_size = input(prompt('value', 'batch size', skip=True, arg='128'))
+    if batch_size == '':
```

Comment:
you shouldn't need these if statements, because the default should be set in prompt_value()